akka-remote/src/main/scala/akka/remote/Remoting.scala

```diff
@@ -452,7 +452,9 @@ private[remote] class EndpointManager(conf: Config, log: LoggingAdapter) extends
         }
         Stop
 
-      case HopelessAssociation(localAddress, remoteAddress, Some(uid), _) ⇒
+      case HopelessAssociation(localAddress, remoteAddress, Some(uid), reason) ⇒
+        log.error(reason, "Association to [{}] with UID [{}] irrecoverably failed. Quarantining address.",
+          remoteAddress, uid)
```

Comment:
what is the strategy for logging? isn't this published/logged by the QuarantinedEvent (but I can understand that the `reason` is important to include)